app/controllers/cases_controller.rb

```diff
@@ -234,6 +234,9 @@ def edit
 
   def edit_closure
     authorize @case, :update_closure?
+    if @case.ico?
+      @s3_direct_post = S3Uploader.s3_direct_post_for_case(@case, 'ICO_decision')
```

Comment:
I think `ICO_decision` should be lower case to match the directory that is used in s3 bucket

Reply:
Yep, still working on this PR